src/using-workflow/using-docker-images.md

```diff
@@ -58,12 +58,29 @@ Use `deis scale cmd=3` to increase `cmd` processes to 3, for example. Scaling a
 process type directly changes the number of [Containers][container]
 running that process.
 
-!!! attention
-    Support for Docker registry authentication is coming soon.
+## Private Registry
 
+In order to deploy Docker images from a private repository on to Deis Workflow the following steps
```

Comment:
> To deploy Docker images from a private registry, take the following steps:

* Gather the username and password for the registry, such as a [Quay.io Robot Accounts][] or [GCR.io Long Lived Token][]